google-cloud-storage/lib/google/cloud/storage/file.rb

```diff
@@ -1546,7 +1552,7 @@ def public_url protocol: :https
         def signed_url method: nil, expires: nil, content_type: nil,
```

Comment:
Will scheme be exposed in the surface here?

Reply:
Yes.

Reply:
Added, but PTAL at the docs description and suggest replacement if you have it.